File: python/list-tupla/prueba.py
#!/usr/bin/env python3

def main():
    # Tupla is immutable
    print("TUPLA")
    x = (1,2,3,4)
    for i in x:
        print(i)

    # List is mutable
    print("LISTA")
    y = [1,2,3,4]
    for i in y:
        print(i)

if __name__ == "__main__":
    main()

File: ruby/gsub/prueba.rb
#!/usr/bin/env ruby

print "Cual es tu nombre? "
name = gets.chomp

def clave(name)
  name.downcase.gsub('a','@').gsub(' ','')
  .gsub('e','3').gsub('i','_').gsub('o','0')
  .gsub('u','#').gsub('s','$')
end

puts "Hi there #{name}!"
puts "Tu clave de acceso es: #{clave(name).reverse}"